bitnami/common/README.md

```diff
@@ -0,0 +1,159 @@
+# Bitnami Common Library Chart
+
+A [Helm Library Chart](https://helm.sh/docs/topics/library_charts/#helm) for grouping common logic between bitnami charts.
+
+## TL;DR;
+
+```yaml
+dependencies:
+- name: common
+  version: 0.1.0
+  repository: https://charts.bitnami.com/bitnami
+```
+
+```bash
+$ helm dependency update
+```
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "common.names.fullname" . }}
+data:
+  myvalue: "Hello World"
+```
+## Introduction
+
+This chart provide a common template helpers which can be used to develop new charts using [Helm](https://helm.sh) package manager.
+
+Bitnami charts can be used with [Kubeapps](https://kubeapps.com/) for deployment and management of Helm Charts in clusters. This Helm chart has been tested on top of [Bitnami Kubernetes Production Runtime](https://kubeprod.io/) (BKPR). Deploy BKPR to get automated TLS certificates, logging and monitoring for your applications.
+
+## Prerequisites
+
+- Kubernetes 1.12+
+- Helm 2.16+ or Helm 3.0-beta3+
+- PV provisioner support in the underlying infrastructure
+- ReadWriteMany volumes for deployment scaling
+
+## Helpers
```

Comment:
@carrodher correct me if I'm wrong but I think that some automation we're doing expects this section to be called `## Parameters`. Am i right?

Reply:
There is a tool doing some replaces in the README before send it to marketplaces like IBM or Azure. If we are going to use this chart as a dependency for other charts we will need to send it to the Index of those companies, but I am not sure if the chart itself is going to be listed in the marketplaces as it is not installable by itself.

So, if you can use `## Parameters` is better, although we need to study this user case regarding marketplaces publishing